stdlib/public/SDK/ObjectiveC/ObjectiveC.swift

```diff
@@ -196,10 +196,10 @@ typealias Zone = NSZone
 //===----------------------------------------------------------------------===//
 
 @warn_unused_result
-@_silgen_name("objc_autoreleasePoolPush")
+@_silgen_name("_swift_objc_autoreleasePoolPush")
```

Comment:
It does not look like these functions are declared in the SDK, so there's no header that we could use.